### PR TITLE
Adding Cocktail-sort

### PR DESCRIPTION
**Pull request by TechnoPhasePRO**
**Fixes issue:**#4644

Code for cocktail sort is added in python.

**Changes:**

### CODE REVIEW

File: code/sorting/src/cocktail_sort/cocktail_sort.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# cocsort is the function used to sort array
+# it is just like bubble sort with less complexity comparatively
+def cocsort(l):
```
Comment (arnavb) [inline code/sorting/src/cocktail_sort/cocktail_sort.py]: Rename this function to `cocktail_sort`. Make sure to update the comment above. Also, if you have not already, reformat this file using Black.

File: code/sorting/src/cocktail_sort/cocktail_sort.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            if l[i] < l[i + 1]:
+                continue
+            else:
+                temp = l[i]
+                l[i] = l[i + 1]
+                l[i + 1] = temp
+                # t is set to 1 to show array has completed one backward sort pass.
+                t = 1
+        # value of first variable is increased everytime as smallest element is found after backward sort
+        first = first + 1
```
Comment (arnavb) [inline code/sorting/src/cocktail_sort/cocktail_sort.py]:
```suggestion
        first += 1
```

**Comment by TechnoPhasePRO** [comment]
@arnavb sir, i have made changes in new code. Please review my PR.

**Comment by TechnoPhasePRO** [comment]
@arnavb sir please give Gssoc20 labels to my merged PRs.

**Comment by TechnoPhasePRO** [comment]
@arnavb sir please give Gssoc20 labels to my merged PRs. My score is still not updating due to this.

**Comment by TechnoPhasePRO** [comment]
@arnavb Sir, Please provide label of GSSoC2020 to my merged PRs.

**Comment by TechnoPhasePRO** [comment]
@arnavb Sir, please label GSSoC20 to this merged PR. My score hasn't updated due to this.

**Comment by TechnoPhasePRO** [comment]
@arnavb sir please give labels and assigned this task to me as I am not getting scores for this merged PRs.

**Pull request merged**